test/testFunctions.js

```diff
@@ -205,56 +303,70 @@ const checkUserReceivesTulipTokens = async (ST, BT, user) => {
 
   const aucIdx = await getAuctionIndex()
   const [returned, tulips] = (await dx.claimBuyerFunds.call(ST.address, BT.address, user, aucIdx)).map(amt => amt.toNumber())
+  const amtClaimed = (await dx.claimedAmounts.call(ST.address, BT.address, aucIdx, user)).toNumber()
   // set global tulips state
   console.log(`
-    RETURNED  = ${returned}
-    TULIPS    = ${tulips}
+    RETURNED          = ${returned.toEth()}           <-- Current amt returned in this fn call
+    AMOUNT(S) CLAIMED = ${amtClaimed.toEth()}         < -- THIS + RETURNED = TULIPS
+
+    TULIPS            = ${tulips.toEth()}             <-- Accumulation of returned + claimedAmounts
+  `)
+  let newBalance = (await dx.balances.call(ST.address, user)).toNumber()
+  console.log(`
+    USER'S ETH AMT = ${newBalance.toEth()}
   `)
-  assert.equal(returned, tulips, 'for ETH -> * pair returned tokens should equal tulips minted')
+  assert.equal(returned + amtClaimed, tulips, 'for ETH -> * pair returned tokens should equal tulips minted')
 
   /*
      * SUB TEST 3: CLAIMBUYERFUNDS - CHECK BUYVOLUMES - CHECK LOCKEDTULIPS AMT = 1:1 FROM AMT IN POSTBUYORDER
      */
   const { receipt: { logs } } = await claimBuyerFunds(ST, BT, false, false, user)
   console.log(logs ? '\tCLAIMING FUNDS SUCCESSFUL' : 'CLAIM FUNDS FAILED')
-  console.log(logs)
+  // console.log(logs)
 
   const buyVolumes = (await dx.buyVolumes.call(ST.address, BT.address)).toNumber()
   console.log(`
-    CURRENT ETH//GNO bVolume = ${buyVolumes}
+    CURRENT ETH//GNO bVolume = ${buyVolumes.toEth()}
   `)
 
   const tulFunds = (await tokenTUL.balanceOf.call(user)).toNumber()
   const lockedTulFunds = (await tokenTUL.getLockedAmount.call(user)).toNumber()
   // set global state
   // userTulips = lockedTulFunds
-  const newBalance = (await dx.balances.call(ST.address, user)).toNumber()
+  newBalance = (await dx.balances.call(ST.address, user)).toNumber()
   console.log(`
-    USER'S OWNED TUL AMT = ${tulFunds}
-    USER'S LOCKED TUL AMT = ${lockedTulFunds}
+    USER'S OWNED TUL AMT  = ${tulFunds.toEth()}
+    USER'S LOCKED TUL AMT = ${lockedTulFunds.toEth()}
 
-    USER'S ETH AMT = ${newBalance}
+    USER'S ETH AMT = ${newBalance.toEth()}
   `)
-  // due to passage of time(stamp)
-  assert.isAtLeast(lockedTulFunds, tulips, 'final tulip tokens are slightly > than calculated from dx.claimBuyerFunds.call')
+  // with changes, TULIPS are NOT minted until auctionCleared
+  // lockedTulFunds should = 0
+  assert.isAtLeast(lockedTulFunds, 0, 'final tulip tokens are slightly > than calculated from dx.claimBuyerFunds.call')
   assert.isAtLeast(newBalance, lockedTulFunds, 'for ETH -> * pair returned tokens should equal tulips minted')
 }
 
 /**
  * unlockTulipTokens
  * @param {address} user => address to unlock Tokens for
  */
-const unlockTulipTokens = async (user) => {
+const unlockTulipTokens = async (user, amt = undefined) => {
   const { TokenTUL: tokenTUL } = await getContracts()
```

Comment:
`amt = undefined` doesn't do a thing. `amt` will be `undefined` if it's not passed anyway.
Also it's never used in the function body

Reply:
@Velenir done